aiohttp/helpers.py

```diff
@@ -402,8 +408,13 @@ def log(self, message, environ, response, transport, time):
         :param float time: Time taken to serve the request.
         """
         try:
-            self.logger.info(self._log_format % self._format_line(
-                [message, environ, response, transport, time]))
+            fmt_info = self._format_line(
+                [message, environ, response, transport, time])
+
+            extra = {name: value for name, value in fmt_info}
+            self.logger.info(self._log_format %
+                             tuple([value for _, value in fmt_info]),
```

Comment:
not currently as fmt_info is a tuple of tuples to keep items in format order. Would you prefer it be a OrderedDict/namedtuple?